docs/tutorial/projects.md

```diff
@@ -222,8 +222,20 @@ A few important notes on what we're looking at here:
 
     For more examples of configuration options available for cloning projects, see [the `git_clone_project` step documentation](/api-ref/prefect/projects/steps/pull).
```

Comment:
I think the private repo example should be inside the `prefect.yaml` callout and before "For more examples..."